allergy intolerances/README.md

```diff
@@ -1 +1,168 @@
+        "clinicalStatus": "active",
+        "verificationStatus": "unconfirmed",
+        "category": [
+            "medication"
+        ],
+        "code": {
+            "coding": [
+                {
+                    "extension": [
+                        {
+                            "url": "https://fhir.nhs.uk/STU3/StructureDefinition/Extension-coding-sctdescid",
+                            "extension": [
+                                {
+                                    "url": "descriptionId",
+                                    "valueId": "1488801013"
+                                },
+                                {
+                                    "url": "descriptionDisplay",
+                                    "valueString": "H/O: aspirin allergy"
+                                }
+                            ]
+                        }
+                    ],
+                    "system": "http://snomed.info/sct",
+                    "code": "395102008",
+                    "display": "H/O: aspirin allergy"
+                }
+            ],
+            "text": "H/O: aspirin allergy"
+        },
+        "patient": {
+            "reference": "Patient/180b44bf-31d8-407b-b8ca-994a3f4a226c"
+        },
+        "onsetDateTime": "2010-01-13",
+        "assertedDate": "2010-01-13",
+        "recorder": {
+            "reference": "Practitioner/3707E1F0-9011-11EC-B1E5-0800200C9A66"
+        },
+        "asserter": {
+            "reference": "Practitioner/3707E1F0-9011-11EC-B1E5-0800200C9A66"
+        },
+        "note": [
+            {
+                "text": "Drug Allergy - Apsrin"
+            }
+        ]
+    }
+},
+```
+</details>
+
+## JSON FHIR > XML HL7
+
+An Allergy Intolerance is mapped to a CompoundStatement with inner ObservationStatement.
+
+| Mapped to (XML HL7)                                                          | Mapped from (JSON FHIR / other source )                        |
+|------------------------------------------------------------------------------|----------------------------------------------------------------|
+| CompoundStatement / id \[@root]                                              | `AllergyIntolerance.id`                                        |
+| CompoundStatement / code                                                     | `AllergyIntolerance.category[0]`                               |
+| CompoundStatement / effectiveTime                                            | `AllergyIntolerance.onsetDateTime`                             |
+| CompoundStatement / availabilityTime                                         | `AllergyIntolerance.assertedDate`                              |
+| ObservationStatement / id \[@root]                                           | `AllergyIntolerance.id`                                        |
+| ObservationStatement / code                                                  | `AllergyIntolerance.code`                                      |
```

Comment:
The adaptor seems to be mapping the the code element as a null flavour with original text. Is the example below produced by the GP2GP adaptor?